src/com/genome2d/text/GTextureTextRenderer.hx

```diff
@@ -351,9 +351,16 @@ class GTextureTextRenderer extends GTextRenderer {
 					if (wordWrapWhitespace) {
 						var backtrack:Int = i - whiteSpaceIndex - 1;
 						var currentCount:Int = currentLine.length;
+						if (backtrack >= currentCount) {
```

Comment:
instead of rendering nothing when there is not whitespace on current line, try to break a word on last character that wont fit